### PR TITLE
added configuration changes for moving over to Google tag manager

### PR DESCRIPTION
**Pull request by thirlz95**

### CODE REVIEW

**Review by Tateni93 — APPROVED** [review]

File: app/uk/gov/hmrc/nisp/config/ApplicationConfig.scala [hunk 1]
```diff
@@ -66,6 +68,8 @@ object ApplicationConfig extends ApplicationConfig with ServicesConfig {
   override lazy val ssoUrl: Option[String] = configuration.getString(s"portal.ssoUrl")
   lazy val frontendTemplatePath: String = configuration.getString("microservice.services.frontend-template-provider.path").getOrElse("/template/mustache")
 
+  override lazy val googleTagManagerId = loadConfig(s"google-tag-manager.id")
```
Comment (danewatts) [inline app/uk/gov/hmrc/nisp/config/ApplicationConfig.scala:71]: Dont need interpolator

**Review by richardbaconhmrc — APPROVED** [review]
Looks good to me.

File: app/uk/gov/hmrc/nisp/views/main.scala.html [hunk 2]
```diff
@@ -55,13 +58,16 @@
   }
 }
 
-@scriptElement = @{
+@scriptElement = {
+  @if(ApplicationConfig.isGtmEnabled) {
+    @views.html.google_tag_manager(ApplicationConfig.googleTagManagerId)
+  }
     Seq(
```
Comment (danewatts) [inline app/uk/gov/hmrc/nisp/views/main.scala.html:65]: Alignment

Comment (danewatts) [reply app/uk/gov/hmrc/nisp/views/main.scala.html:65]: Does this work? Not sure how twirl will render this but scala would only return the last thing on the page which is the `Seq[Map[_,_]]`

**Review by danewatts — CHANGES_REQUESTED** [review]

File: app/uk/gov/hmrc/nisp/views/main.scala.html (hunk 2 above)
Comment (matthew-mcnally) [reply app/uk/gov/hmrc/nisp/views/main.scala.html]: It does seem to be getting different values from the Seq

**Review by matthew-mcnally — APPROVED** [review]

**Pull request merged**